todo_app/tests_e2e/test_selenium.py

```diff
@@ -0,0 +1,127 @@
+import os
+import pytest
+import requests
+
+from todo_app import app, data
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+BOARD_URL = "https://api.trello.com/1/boards/"
+BOARDS_URL = "https://api.trello.com/1/members/me/boards?fields=name,url"
+
+KEY=os.environ.get('TRELLO_API_KEY')
```

Comment:
These environment variables are set when the test is created, but depend on your system already having the correct environment variables loaded in, which means this fails if the tests are run manually using e.g. `poetry run pytest`

Loading the `.env` file before these lines, in an equivalent way to loading the `.env.test` that you did in the integration test, should resolve that

Reply:
Suggested changes have been made. Thanks